CHANGELOG.md

```diff
@@ -1,8 +1,5 @@
 # nf-core/mhcquant: Changelog
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
-
```

Comment:
May want to put a `## dev` header in here ([see](https://github.com/nf-core/atacseq/blob/dev/CHANGELOG.md)) for changes made in the `dev` version. Also, worth adding a note that you have updated to tools 1.9

Reply:
https://github.com/Zethson/MHCquant/commit/054a68542f25314188e860dc26106f350f771000